cylc/flow/scheduler.py

```diff
@@ -164,6 +164,29 @@ class Scheduler:
         'reload_suite'
     )
 
+    # managers
+    profiler: Profiler
+    pool: TaskPool
+    proc_pool: SubProcPool
+    task_job_mgr: TaskJobManager
+    task_events_mgr: TaskEventsManager
+    suite_event_handler: SuiteEventHandler
+    data_store_mgr: DataStoreMgr
+    suite_db_mgr: SuiteDatabaseManager
+    broadcast_mgr: BroadcastMgr
+    xtrigger_mgr: XtriggerManager
```

Comment:
If making these attributes without defaults, they have to be before attributes with defaults in an `@dataclass` class